src/DacpacTool/ScriptParser.cs

```diff
@@ -69,7 +69,15 @@ BatchParserAction ICommandHandler.Include(TextBlock includeBlock, out TextReader
             if (!Path.IsPathRooted(includedFileName))
             {
                 var position = lineInfo.GetStreamPositionForOffset(0);
-                includedFileName = Path.Combine(Path.GetDirectoryName(position.Filename), includedFileName);
+
+                var projectDir = Path.GetDirectoryName(position.Filename);
```

Comment:
If we're saying `Unable to resolve path for included file`, I think we should test the file itself, instead of the directory. Couldn't we have a scenario here where we validate that `projectDir` exists, but `includedFileName` _doesn't_ exist? Or am I misunderstanding what this is checking?